genlocalnetappbins.sh

```diff
@@ -3,7 +3,7 @@ set -euo pipefail
 DOCKERFILE="${1:-Dockerfile-build}"
 IMAGE_TAG="${2:-dcld-build}"
 VERSION_DIR="${3:-genesis}"
-LATEST_VERSION="${4:-false}"
+if [[ "${4}" == "v0.13.0-pre" ]]; then LATEST_VERSION="true"; else LATEST_VERSION="false"; fi
```

Comment:
It's not good to specify the magic constant "v0.13.0-pre" here again, and I think we don't need it.
I think the parameter LATEST_VERSION can be renamed to MAINNET_STABLE_VERSION, and can be either "v0.12.0" (passed from Makefile) or empty. MAINNET_STABLE_VERSION can be used in line 14 instead of v0.12.0.
v0.12.0 constant can be moved to Makefile.test.